addons/sale_pdf_quote_builder/models/product_document.py

```diff
@@ -1,8 +1,12 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+import base64
+
 from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
 
+from odoo.addons.sale_pdf_quote_builder import utils
```

Comment:
Why not with other import of odoo?

Reply:
Because it's import from an addon module, they are separate =]